specification/loadtestservice/LoadTestService/tspconfig.yaml

```diff
@@ -19,7 +20,8 @@ options:
   #   azure-resource-provider-folder: "data-plane"
   #   examples-directory: ./examples
   "@azure-tools/typespec-python":
-    package-name: "azure-developer-loadtesting"
+    package-dir: "azure-developer-loadtesting"
```

Comment:
Why is this being added here? Where else would it be used aside from the package name?

Reply:
@raych1 @msyyc Could you help on this?

Reply:
@catalinaperalta The config item has been decided by @weshaggard and SDK automation script is updated so the change is expected. Here is template: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml